src/main/game/Board.java

```diff
@@ -46,96 +57,84 @@ public boolean hasAvailableMoves() {
         int count = 0;
 
         for (String space : this.grid) {
-            if (space.equals(PLAYER_ONE_MARK.getPlayer()) || space.equals(PLAYER_TWO_MARK.getPlayer())) {
+            if (space.equals(X_MARK.getPlayer()) || space.equals(O_MARK.getPlayer())) {
```

Comment:
Extracting this into a method like `cellTaken` would make this easier to read 👍🏼